scripts/prep.sh

```diff
@@ -13,6 +13,13 @@ if [ $# -ne 3 ]; then
     exit 1
 fi
 
+OLD_STACK_NAME=$1
+NEW_STACK_NAME=$2
+BLOCKCHAIN_PROVIDER=$3
+
+JOBS="msg_broadcast msg_private"
```

Comment:
Do we want to include minting?

Reply:
It gets added here, only if the blockchain provider is `geth`
https://github.com/hyperledger/firefly-perf-cli/blob/188dee9d2992743ac6c66e05a1b85a26e483efdf/scripts/prep.sh#L66